tests/test_l10n_download_links.py

```diff
@@ -6,8 +6,6 @@
 import pytest
 import requests
 
-pytestmark = [pytest.mark.nondestructive, pytest.mark.skip_selenium]
-
 
 def pytest_generate_tests(metafunc):
     base_url = metafunc.config.option.base_url
```

Comment:
Ah yes, I was aware of this issue. I'll revisit and see if there's a way for us to get the value from the fixture. Thanks.